gold-phone-input.html

```diff
@@ -157,9 +157,27 @@
       }
     },
 
-    observers: [
-      '_computeValue(value)'
-    ],
+    listeners: {
+      'input': '_onInput'
+    },
+
+    ready: function() {
+      this.addEventListener('blur', function() {
+        this._handleAutoValidate();
+      }, true);
+
+      // If there's an initial input, validate it.
+      if (this.value)
+        this._handleAutoValidate();
+    },
+
+    /**
+     * If `autoValidate` is true, then validates the element.
+     */
+    _handleAutoValidate: function() {
```

Comment:
This method is essentially copied across all four PRs. Does it belong higher up the hierarchy, in a behavior?